src/core/ui/forms/MarkdownInputControls/MarkdownInputControls.tsx

```
@@ -181,3 +191,23 @@ const MarkdownInputControls = memo(
+  visible?: boolean;
+  editor: Editor | null;
+  hideImageOptions?: boolean;
+  temporaryLocation?: boolean;
+  onDialogOpen?: () => void;
+  onDialogClose?: () => void;
+};
+
+interface ControlsButtonProps extends IconButtonProps {
+  editor: Editor | null;
+  command: (commandsChain: ChainedCommands) => ChainedCommands;
+  specs?: string | [attributes: {}] | [nodeOrMark: string, attributes?: {}];
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)